tests/Resources/ti.platform.test.js

```diff
@@ -225,6 +231,13 @@ describe('Titanium.Platform', function () {
 
 	it('.macaddress', () => {
 		should(Ti.Platform).have.readOnlyProperty('macaddress').which.is.a.String();
+		if (OS_IOS) {
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.platform.test.js;
  Line: 234;
-->
- :no_entry_sign: tests/Resources/ti.platform.test.js line 234 – 'OS_IOS' is not defined. (no-undef)